scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -97,43 +171,73 @@ class ThemeAggregator {
    * Dependency files will resolve to the node_modules directory.
    * Local files will resolve relative to the expected output directory.
    * @param {string} filePath - A file path.
-   * @returns {string} - A resolved file path.
+   * @returns {string} - Resolved file path relative to the home directory or node module path.
    */
   static resolve(filePath) {
+    // Constructs the relative path.
+    const relativePath = path.relative(OUTPUT_PATH, path.resolve(OUTPUT_PATH, filePath));
     if (filePath.indexOf(NODE_MODULES) > -1) {
-      return filePath.substring(filePath.indexOf(NODE_MODULES) + NODE_MODULES.length);
+      const dependencyPath = filePath.substring(filePath.indexOf(NODE_MODULES) + NODE_MODULES.length);
+      return dependencyPath;
     }
 
-    // Constructs the relative path.
-    const outputPath = path.resolve(process.cwd());
-    return `./${path.relative(outputPath, path.resolve(process.cwd(), filePath))}`;
+    return relativePath;
   }
 
   /**
    * Validates the aggregated options.
    * @param {Object} options - The aggregated options.
+   * @returns {boolean} - Whether the theme config has valid options.
    */
   static validate(options) {
     const { theme, scoped } = options;
 
     if (!theme && !scoped) {
       Logger.warn('No theme provided.');
+      return false;
     }
+
+    return true;
   }
 
   /**
-   * Writes a file containing theme imports.
-   * @param {string[]} imports - An array of files to import.
+   * Generates a theme scss file and outputs it to the generatedThemes directory.
+   * @param {string} assets - The aggregated theme files to import within generated file.
+   * @param {string} themeName - Name of theme to aggregate.
+   * @param {string} prefix - Prefix to append to generated file.
+   * @param {string} scopeSelector - scss scope selector to encase theme.
+   * @returns {string} - The path of the generated scss file, relative to the working home directory.
+   */
+  static writeSCSSFile(assets, themeName, prefix, scopeSelector) {
```

Comment:
consider taking in the arguments as an object. I find it's easier to call functions with lots of params when the params are named.
```javascript
writeSCSSFile({
  assets,
  themeName: 'my favorite theme',
  prefix: 'herp',
  scopeSelector: 'root',
});
```

```suggestion
  static writeSCSSFile({assets, themeName, prefix, scopeSelector}) {
```

Reply:
sweet, updated https://github.com/cerner/terra-toolkit/pull/324/commits/0338b825b396b715fba0932250f405d1d6a4556f